.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,205 @@
+name: run-tests
```

Comment:
This runs all the tests, and it has parameters to let us make matrices for node versions. So we could easily create a daily smoke test job that invokes this one the same way the validate_pr one does.